src/v/cluster/feature_table.cc

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "feature_table.h"
+
+#include "cluster/logger.h"
+#include "cluster/types.h"
+
+namespace cluster {
+
+std::string_view to_string_view(feature f) {
+    switch (f) {
+    case feature::central_config:
+        return "central_config";
+    }
+    __builtin_unreachable();
+}
+
+feature_list feature_table::get_active_features() const {
+    // Cast to integer to do lt/gt comparisons
+    auto n_v = static_cast<int64_t>(_active_version);
```

Comment:
I thought it already was signed: `using cluster_version = named_type<int64_t, struct cluster_version_tag>;`